google/cloud/datastore/client.py

```diff
@@ -36,11 +36,14 @@
 except ImportError:  # pragma: NO COVER
     from google.api_core import client_info
 
-    make_datastore_api = None
+    def no_datastore_api(self):
+        return None
+
+    make_datastore_api = no_datastore_api
```

Comment:
Why isn't this just a function?
```suggestion
    def make_datastore_api(self):
        return None
```

And maybe it should raise an exception, since the only reason we're here is that gRPC is disabled?

Reply:
I get it, but if you do that you end up fighting with `google/cloud/datastore/client.py:41: error: All conditional function variants must have identical signatures` :\ This seems not terrible?